contracts/child/ChildValidatorSet.sol

```diff
@@ -115,12 +115,12 @@ contract ChildValidatorSet is
      * @inheritdoc IChildValidatorSetBase
      */
     function commitEpochWithDoubleSignerSlashing(
+        uint256 chainId,
```

Comment:
Instead of passing in the chainId we should use `block.chainid` when constructing the message